src/Scaffold/Console/CreateController.php

```diff
@@ -37,13 +37,32 @@ class CreateController extends GeneratorCommand
         'controller/_list_toolbar.stub' => 'controllers/{{lower_name}}/_list_toolbar.htm',
         'controller/config_form.stub'   => 'controllers/{{lower_name}}/config_form.yaml',
         'controller/config_list.stub'   => 'controllers/{{lower_name}}/config_list.yaml',
-        'controller/create.stub'        => 'controllers/{{lower_name}}/create.htm',
         'controller/index.stub'         => 'controllers/{{lower_name}}/index.htm',
-        'controller/preview.stub'       => 'controllers/{{lower_name}}/preview.htm',
-        'controller/update.stub'        => 'controllers/{{lower_name}}/update.htm',
         'controller/controller.stub'    => 'controllers/{{studly_name}}.php',
     ];
 
+    /**
+     * Execute the console command.
+     *
+     * @return bool|null
+     */
+    public function handle()
+    {
+        /**
+         * Add the form stubs according to the selected layout
+         */
+        $layout = $this->option('layout');
```

Comment:
This should probably be `--form-layout` instead of `--layout` for future improvements.

Reply:
I have changed this as well!